src/base/VolumeInterface.php

```diff
@@ -55,9 +55,28 @@ public function getFileList(string $directory, bool $recursive): array;
      * @param string $uri URI to the file on the volume
      * @return array
      * @throws VolumeObjectNotFoundException if the file cannot be found
+     * @deprecated in Craft 3.6.0. Use [[getFileSize]] and [[getDateModified]] instead.
      */
     public function getFileMetadata(string $uri): array;
 
+    /**
+     * Return the file size.
+     *
+     * @param string $uri
+     * @return mixed
```

Comment:
Could this be more specific (`int|null`)? If so also add a `: ?int` return type declaration to the end.